src/server/saved_objects/service/lib/repository.js

```diff
@@ -210,6 +211,54 @@ export class SavedObjectsRepository {
     );
   }
 
+  /**
+   * Summarize
```

Comment:
In the spirit of the rest of this file, can we have a summary/description of this method?